tests/scripts/task_python_docs.sh

```diff
@@ -81,6 +81,8 @@ IGNORED_WARNINGS=(
     'gen_gallery extension is not safe for parallel'
     'strategy:conv2d NHWC layout is not optimized for x86 with autotvm.'
     'strategy:depthwise_conv2d NHWC layout is not optimized for x86 with autotvm.'
+    'strategy:depthwise_conv2d with layout NHWC is not optimized for arm cpu.'
+    'strategy:dense is not optimized for arm cpu.'
```

Comment:
Curious, does this mean the `arm_cpu` key wasn't tested before this PR?

Reply:
yip, my guess is people didn't know to put `-device=arm_cpu` or `-keys=arm_cpu` so they were using x86 schedules 🙀